docker-compose.yml

```diff
@@ -8,6 +8,7 @@ services:
       args:
         GIT_COMMIT: ${GIT_COMMIT-local-dev}
         PIPENV_DEV: "True"
+    command: "echo 'Base Image Should Not Run'"
```

Comment:
Is this command really necessary? For me, this image never runs locally.

Reply:
On fedora, it stays running as a second koku-server.